inc/ticketrecurrent.class.php

```diff
@@ -372,8 +372,36 @@ function computeNextCreationDate($begin_date, $end_date, $periodicity, $create_b
       }
 
       $calendar = new Calendar();
-      if ($calendars_id && $calendar->getFromDB($calendars_id) && $calendar->hasAWorkingDay()) {
-         // Base computation on calendar if calendar is defined
+      $is_calendar_valid = $calendars_id && $calendar->getFromDB($calendars_id) && $calendar->hasAWorkingDay();
+
+      if (!$is_calendar_valid || $periodicity_in_seconds > DAY_TIMESTAMP) {
```

Comment:
```suggestion
      if (!$is_calendar_valid || $periodicity_in_seconds >= DAY_TIMESTAMP) {
```

Shouldn't this also be applied if the duration is exactly one day ?

Reply:
Indeed. Fixed.